include/SFML/Window/ContextSettings.hpp

```diff
@@ -76,6 +81,8 @@ struct ContextSettings
     unsigned int majorVersion;      ///< Major number of the context version to create
     unsigned int minorVersion;      ///< Minor number of the context version to create
     Uint32       attributeFlags;    ///< The attribute flags to create the context with
+    std::string  vendorName;        ///< Name of the vendor of the OpenGL implementation
+    std::string  rendererName;      ///< Name of the OpenGL renderer implementation
```

Comment:
I wouldn't use the word "select" :wink:. They are more like "hints" for SFML to select a configuration that best suits the requested settings.

If you want, I can add a note to the documentation that states that these fields are not taken into consideration when creating the context.